modules/network-observability-operator-install.adoc

```diff
@@ -34,17 +34,13 @@ This documentation assumes that your `LokiStack` instance name is `loki`. Using
 . Navigate to the *Flow Collector* tab, and click *Create FlowCollector*. Make the following selections in the form view:
 .. *spec.agent.ebpf.Sampling*: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", `spec.agent.ebpf`.
 .. If you are using Loki, set the following specifications:
-... *spec.loki.enable*: Select the check box to enable storing flows in Loki.
-... *spec.loki.url*: Since authentication is specified separately, this URL needs to be updated to `https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network`. The first part of the URL, "loki", must match the name of your `LokiStack`.
-... *spec.loki.authToken*: Select the `FORWARD` value.
-... *spec.loki.statusUrl*: Set this to `https://loki-query-frontend-http.netobserv.svc:3100/`. The first part of the URL, "loki", must match the name of your `LokiStack`.
+... *spec.loki.mode*: Set this to `LokiStack` mode.
```

Comment:
If I understood correctly from the Slack discussion, we're only documenting `LokiStack` mode, right?

Reply:
I have in my notes that `Manual` mode is just the old steps encapsulated in this spec:
 ```
 loki:
    mode: MANUAL
    manual:
      ingesterUrl: 'https://loki-gateway-http.netobserv.svc:8080/api/logs/v1/network/'
      statusUrl: 'https://loki-query-frontend-http.netobserv.svc:3100/'
      authToken: FORWARD
      tls:
        enable: true
        caCert:
          type: configmap
          name: loki-gateway-ca-bundle
          certFile: service-ca.crt
        insecureSkipVerify: false
      statusTls:
        enable: true
        caCert:
          certFile: service-ca.crt
          name: loki-ca-bundle
          type: configmap
        insecureSkipVerify: false
        userCert:
          certFile: tls.crt
          certKey: tls.key
          name: loki-query-frontend-http
          type: secret
 
```
So I agree, we don't need to go into these details I don't think. Unless we find more customers wanting complex Loki installation steps.

Reply:
Does this need documentation? Jw how customers would use these microservices? https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowCollector.md#flowcollectorspeclokimicroservices

Reply:
Correct @skrthomas, we don't need to mention other modes here but it could be more detailed in `Troubleshooting Network Observability` for example.
@jotak would that work with your initial suggestion ?

As you pointed @skrthomas, everything is already described in [FlowCollector.spec.loki
](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowCollector.md#flowcollectorspecloki-2)

Example for `microservices`;
- set `mode` value to `Microservices`
- fill [FlowCollector.spec.loki.microservices](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowCollector.md#flowcollectorspeclokimicroservices) 
  - urls
  - tenant & tls (optionnal)

Reply:
Yes I agree; so the LokiStack mode should be documented here; I would say for the other modes, including Manual, the CRD doc should be sufficient. Maybe just mention here that other modes are available if users don't use the loki operator or if they want more tweakable install.